docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -1827,14 +1858,14 @@ Here are the steps for this use case:
        - To make it easier to keep the life cycle and/or CI/CD process for the two domains simple and independent.
        - To 'future proof' the new domain so that changes to the original domain's secrets or new domain's secrets can be independent.
 
-   If you're following the `JRF` path through the sample, then you also need to deploy the additional secret referenced by macros in the `JRF` model `RCUDbInfo` clause, plus an `OPSS` wallet password secret. For details about the uses of these secrets, see the [Model in Image]({{< relref "/userguide/managing-domains/model-in-image/_index.md" >}}) user documentation.
+   If you're following the `JRF` path through the sample, then you also need to deploy the additional secret referenced by macros in the `JRF` model `RCUDbInfo` clause, plus an `OPSS` wallet password secret. For details about the uses of these secrets, see the [Model in Image]({{< relref "/userguide/managing-domains/model-in-image/_index.md" >}}) user documentation. Note that we are using RCU prefix `FMW2` for this domain, as the first domain is already using `FMW1`.
```

Comment:
using RCU prefix -> using the RCU prefix
as the first domain -> because the first domain (since/as – don't use to mean for the reason of; use because.)